Source/Libraries/GSF.Web/Model/ModelController.cs

```diff
@@ -97,10 +108,20 @@ public ModelController()
             {
                 SecurityType = "Claims";
 
-                IEnumerable<ClaimAttribute> claimAttributes = typeof(T).GetCustomAttributes<ClaimAttribute>();
+                IEnumerable<ClaimAttribute> claimViewAttributes = typeof(T).GetCustomAttributes<ClaimAttribute>();
+                IEnumerable<ClaimAttribute> claimEditAttributes = typeof(U).GetCustomAttributes<ClaimAttribute>();
 
-                foreach (ClaimAttribute claimAttribute in claimAttributes)
+                foreach (ClaimAttribute claimAttribute in claimViewAttributes)
+                {
+                    if (claimAttribute.Verb != "GET") continue;
```

Comment:
This does not work the way you intended. The `SearchableList` endpoint for instance is `POST` but should be using the `claimViewAttributes`.